.circleci/config.yml

```diff
@@ -47,14 +47,14 @@ jobs:
             - plugins/@grouparoo/hubspot/node_modules
             - plugins/@grouparoo/iterable/node_modules
             - plugins/@grouparoo/mailchimp/node_modules
-            - plugins/@grouparoo/mysql/node_modules
-            - plugins/@grouparoo/onesignal/node_modules
-            - plugins/@grouparoo/pipedrive/node_modules
-            - plugins/@grouparoo/redshift/node_modules
-            - plugins/@grouparoo/salesforce/node_modules
-            - plugins/@grouparoo/sentry/node_modules
-            - plugins/@grouparoo/spec-helper/node_modules
-            - plugins/@grouparoo/zendesk/node_modules
+            - plugins/@grouparoo/mongo/node_modules
```

Comment:
@evantahler  there would be less churn on these if, instead of `even` and `odd` we did something like alphabetize and then each has `length / 2`

Or maybe  set a max size for each (15?). and know the number of buckets (3). and then freak out when there are 4 buckets (>45)

Reply:
Another PR of course